src/services/edusharing/logic/connector.js

```diff
@@ -90,12 +94,12 @@ class EduSharingConnector {
 
 	allConfigurationValuesHaveBeenDefined() {
 		return (
-			Configuration.has('ES_DOMAIN')
-			&& Configuration.has('ES_USER')
-			&& Configuration.has('ES_PASSWORD')
-			&& Configuration.has('ES_GRANT_TYPE')
-			&& Configuration.has('ES_OAUTH_SECRET')
-			&& Configuration.has('ES_CLIENT_ID')
+			Configuration.has('ES_DOMAIN') &&
+			Configuration.has('ES_USER') &&
+			Configuration.has('ES_PASSWORD') &&
+			Configuration.has('ES_GRANT_TYPE') &&
```

Comment:
Codacy found an issue: ['&&' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)